File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    var index = 0;
    while (true) {
        for (var i = index + 1; i < nums.length; i++) {
            if (nums[index] + nums[i] == target) {
                return [index, i];
            }
        }
        index++;
    }
};

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        idx = 0
        while True:
            for num in range(idx + 1, len(nums)):
                if nums[idx] + nums[num] == target:
                    return [idx, num]
            idx += 1